### ISSUE TITLE
Add base timeout functionality

### ISSUE THREAD

**Issue by Tellinq** [Tellinq/Discord-Moderation-Bot#11]
Needs to be able to interact with the API in a compliant manner

We need:
Duration
Reason

**Comment by Tellinq** [comment]
I need this restored in an abstract manner. We had this implemented as a command, unless Discord’s timeout API is extremely straightforward and we end up not needing a base, we need to figure out how this should be done

**Comment by Tellinq** [comment]
@Aetopia thoughts?

**Comment by Aetopia** [comment]
syntax: /timeout <subcommand> <arguments>
subcommands:
add
remove

When adding timeouts, we could have simple presets for a predefined duration instead of specifying the duration manually.

**Comment by nicklvh** [comment]
Arguments would be specifying the length and reason?

**Comment by Tellinq** [comment]
Yeah as a base. Just to clarify I don’t expect to have a command for this. I’d rather have other commands like the /punish command with predetermined reasons and length to do this
